aiohttp/web_runner.py

```diff
@@ -238,6 +242,24 @@ def sites(self) -> Set[BaseSite]:
 
         self._server = await self._make_server()
 
+    async def serve_forever(self) -> None:
+        if self._serving_forever_fut is not None:
+            raise RuntimeError(
+                'Concurrent calls to serve_forever() are not allowed'
+            )
+        if self._server is None:
+            raise RuntimeError('Call setup() first')
+
```

Comment:
Note: [`asyncio.Server.serve_forever()`](https://docs.python.org/3.7/library/asyncio-eventloop.html#asyncio.Server.serve_forever) actually *starts* accepting connection when `serve_forever()` was called. In contrast, current method does not start anything (though it cleans itself up if cancelled).